### PR TITLE
Fiducial

### PR DESCRIPTION
**Pull request by kuntzer**
This branch is now ready for a pull request.

It already has some files linked to a variable PSF that will be reorganised soon.

### CODE REVIEW

**Comment by mtewes** [comment]
Having a look at this now

File: megalut/learn/tenbilacrun.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def predict(catalog, conflist, workbasedir=None):
 		if os.path.isfile(toolconfpath):
 			toolconfig = readconfig(toolconfpath)
 			trainworkdir = os.path.join(workbasedir, dataconfig.get("setup", "name") + "_" + toolconfig.get("setup", "name"))
-		elif os.path.isdir(toolconfpath)
+		elif os.path.isdir(toolconfpath):
```
Comment (mtewes) [inline megalut/learn/tenbilacrun.py]: This was fixed already, I think -- maybe remerge the latest master into the branch ?

File: runs/great3/old/mysimparams.py [hunk 2]
```diff
@@ -1,177 +0,0 @@
-"""
```
Comment (mtewes) [inline runs/great3/old/mysimparams.py]: Maybe let's keep these files ?

All the rest is fine (I checked only the "package" changes), OK for merge!

File: megalut/learn/tenbilacrun.py (hunk 1 above)
Comment (kuntzer) [inline megalut/learn/tenbilacrun.py]: Not on what's pushed on the master, no

Comment (mtewes) [inline megalut/learn/tenbilacrun.py]: Ideed, forgot to commit this. Sorry!
So all is fine, ready for merging!

File: runs/great3/old/mysimparams.py (hunk 2 above)
Comment (kuntzer) [inline runs/great3/old/mysimparams.py]: Okay, they're added

**Comment by kuntzer** [comment]
Leaving the fiducial branch alive...

**Pull request merged**